examples/README.md

```diff
@@ -21,19 +20,21 @@ setup. See [this guide][cert-operator-local-setup] for details. The operator and
 to be used during aws-operator setup can be installed with:
 
 ```bash
+helm registry install quay.io/giantswarm/vaultlab-chart:stable -- \
```

Comment:
Is the `--` here needed?

Reply:
In this case it's not strictly required but as soon as we add an option to registry it would be. It's a way to keep separate the options related to the registry plugin from those related to the helm command itself. From `helm registry install --help`:
```
Fetch a Chart from the app-registry and execute `helm install`.
Helm's options can be passed on the command:
$ appr helm install [APPR_OPTS] -- [HELM_OPTS]
$ appr helm install quay.io/ant31/cookieapp -- --set imageTag=v0.4.5 --namespace demo
```
Although we could remove them in this case I'd rather keep it, WDYT?

Reply:
Got it thanks. Yes I'm fine with keeping them.